src/functions-reference/distributions_over_unbounded_vectors.Rmd

```diff
@@ -593,6 +594,139 @@ freedom nu, locations mu, and scale matrix Sigma; may only be used in
 transformed data andgenerated quantities blocks
 `r since("2.18")`
 
+## Multivariate Student-t distribution, Cholesky parameterization {#multi-student-t-cholesky-fun}
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in \mathbb{R}^+$, $\mu \in \mathbb{R}^K$,
+and $\Sigma \in \mathbb{R}^{K \times K}$ is symmetric and positive
+definite, then for $y \in \mathbb{R}^K$, \[ \begin{array}{l}
+\text{MultiStudentTCholesky}(y\,\mid \nu,\,\mu,\,L) \\  =
+\frac{1}{\pi^{K/2}} \ \frac{1}{\nu^{K/2}} \ \frac{\Gamma\!\left((\nu +
+K)/2\right)}      {\Gamma(\nu/2)} \ \frac{1}{\left| L
+\right|} \ \left( 1 + \frac{1}{\nu} \, \left(y - \mu\right)^{\top} \,
+L^{-T}L^{-1} \, \left(y - \mu\right) \right)^{-(\nu + K)/2} \! .
+\end{array} \] If $L$ is lower triangular and
+$LL^{top}$ is a $K \times K$ positive definite matrix, then $L_{k,k}$
+must be strictly positive for $k \in 1{:}K$.  If an $L$ is provided
```

Comment:
remove---raising errors isn't part of the math, but part of the functions.  the math's just a function defined when things satisfy constraints